src/Bicep.Cli.IntegrationTests/BuildParamsCommandTests.cs

```diff
@@ -351,27 +351,6 @@ public async Task Build_Invalid_Single_Params_File_ShouldFail_WithExpectedErrorM
             }
         }
 
-        [TestMethod]
-        [EmbeddedFilesTestData(@"Files/BuildParamsCommandTests/.*/main\.bicepparam")]
-        [TestCategory(BaselineHelper.BaselineTestCategory)]
-        public async Task Build_params_to_stdout_with_non_bicep_references_should_succeed(EmbeddedFile paramFile)
```

Comment:
Did you mean to remove this test?

Reply:
no, it was a mistake, reverted it back 👍